src/theme.js

```diff
@@ -336,7 +336,7 @@ const sharedStyles = {
       },
 
       '& li': {
-        marginTop: 20,
+        marginTop: 10,
```

Comment:
Reduces spacing between list elements:

After:
![image](https://user-images.githubusercontent.com/1571667/37690849-f490b74c-2c6a-11e8-9f53-12f13e8e285a.png)

Before:
![image](https://user-images.githubusercontent.com/1571667/37690882-14b77f10-2c6b-11e8-9dc0-3059686d124c.png)

Reply:
I think it looks nice.